makefiles/Makefile.docs-ruby

```diff
@@ -59,7 +59,7 @@ get-build-dependencies: fetch-submodule
 
 fetch-submodule:
 	git submodule update --remote --init
-	rsync -a --delete ${REPO_DIR}/mongo-ruby-driver/docs/ ${REPO_DIR}/source
+	-rsync -a --delete ${REPO_DIR}/mongo-ruby-driver/docs/ ${REPO_DIR}/source
 	rsync -a --delete ${REPO_DIR}/bson-ruby/docs/tutorials/bson.txt ${REPO_DIR}/source/tutorials/bson.txt
```

Comment:
(Non-blocking wonder) Looked at the ticket, but I wonder why we're interested in removing only the `mongo-ruby-driver` submodule instead of both 🤔

Reply:
It's a good question! I'll ask!

Reply:
We are removing both!

Reply:
Sounds good! Thanks for double-checking! LGTM!